### PR TITLE
🎨 UI improvements

### PR DESCRIPTION
**Pull request by jagodarybacka**
Resolves https://github.com/tallycash/extension/issues/2276
Resolves https://github.com/tallycash/extension/issues/2279
Resolves https://github.com/tallycash/extension/issues/2278

### What

Various small fixes:
- don't allow dapp title and URL to overflow 
image
image
- fix back button on Conencted websites and other subpages of Settings
- rename Overview to Portfolio

### CODE REVIEW

File: ui/pages/Claiming/Eligible.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export default function Eligible(): ReactElement {
   }
 
   if (Object.keys(accountData).length === 0) {
-    return <Redirect to="/overview" />
```
Comment (mhluongo) [inline ui/pages/Claiming/Eligible.tsx:63]: If someone is on the `/overview` route and this update landed, what happens?

**Review by mhluongo — COMMENTED** [review]
Excited about this! Quick q on route persistence

File: ui/pages/Claiming/Eligible.tsx (hunk 1 above)
Comment (jagodarybacka) [reply ui/pages/Claiming/Eligible.tsx:63]: The unknown route should redirect to root so "wallet" page. This was fixed couple weeks ago.

**Review by mhluongo — APPROVED** [review]
Bellisimo :ok_hand:

**Pull request merged**